apps/src/templates/currentUserRedux.js

```diff
@@ -33,7 +37,8 @@ const initialState = {
   userName: null,
   userType: 'unknown',
   signInState: SignInState.Unknown,
-  hasSeenStandardsReportInfo: false
+  hasSeenStandardsReportInfo: false,
+  under13: true
```

Comment:
This default makes sense for our scenario, and likely others, but might be worth adding a comment here explaining why we default to `true`.